### DISCUSSION TITLE
Figma UI kit - Single vs Team

### DISCUSSION THREAD

**Discussion by MichelaFortuna** [primefaces/community#1132] [Figma UI Kit]
Hi, I currently work for a company in which I am the only designer and I have a couple of questions:

-I was thinking about purchasing the Figma UI kit for a single designer - would that limit in any way the access to it for the developers or other team members in the future? Or is it just the regular .fig file that I can download and use?
-Are you including updates in the Figma UI kit? Or would I have to buy the kit again in the case any update is done on your side?

Thanks  

M Fortuna

**Comment by onursenture** [comment]
Hi, 

You can purchase the 'Single Designer' license if you are a solo designer. When you buy the product and import the acquired .fig file, you can share the resulting project with developers and other team members.

There is no need to repurchase the UI Kit for updates.
